tests/samples/test_api.py

```diff
@@ -834,14 +834,26 @@ async def test_find_analyses(
         {"_id": "foo", "name": "Malus domestica", "nickname": "Apple"}
     )
 
+    job_1 = await fake2.jobs.create(user=user_1)
+
+    job_2 = await fake2.jobs.create(user=user_2)
+
+    await client.db.references.insert_many(
+        [
+            {"_id": "foo", "data_type": "genome", "name": "Foo"},
+            {"_id": "baz", "data_type": "genome", "name": "Baz"},
+        ],
+        session=None,
+    )
+
     await client.db.analyses.insert_many(
```

Comment:
Need a test for when the `job.id` field is set in the analysis, but no corresponding job is found in the database. This should result in the Pydantic field being set `None`.